src/_tests_/components/__snapshots__/forecast-details.test.js.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
```

Comment:
In your test file you need to call asFragment() in your expect block. We shouldn't be seeing exports[`ForecastDetails renders correctly 1`] = `[Function]`;

There should be a lot more code. But don't worry this was a typo in the track, so you didn't do anything correctly and we have now updated it :)